File: app/src/main/java/com/kronos/sample/DateHelper.java
package com.kronos.sample;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class DateHelper {
    public static String getTime(long timeInMillis, SimpleDateFormat dateFormat) {
        return dateFormat.format(new Date(timeInMillis));
    }


    public static String getTime(long timeInMillis, String format) {
        SimpleDateFormat dateFormat = new SimpleDateFormat(format, Locale.CHINA);
        return getTime(timeInMillis, dateFormat);
    }
}

File: app/src/main/java/com/kronos/sample/DateHelper.kt
package com.kronos.sample

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

fun Long.getTime(dateFormat: SimpleDateFormat): String {
    return dateFormat.format(Date(this))
}

fun Long.getTime(format: String): String {
    val dateFormat = SimpleDateFormat(format, Locale.CHINA)
    return getTime(dateFormat)
}

